_posts/2022-09-04-edge-example-app-sense-the-internet-part-1.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title:  "Edge Example App: Sense the Internet Part 1"
+date:   2022-09-04 03:27:05 +0100
+categories: flotta
+author: Ahmad Ateya <ahmad.m.ateya@gmail.com>
+tags:
+- flotta
+- devices
+- workloads
+
+summary-1:
+---
+Edge Example App is an app for Flotta Edge devices, with a workload that will be deployed on the device that has 2 main features:
+- Sensing the Internet (which helps to construct devices network topology). <= this article
+- Read CPU temperature (which indicate how much load the device is handling).
+
+As Project Flotta goal is to manage workloads deployed on small-footprint devices and dealing with network connectivity issues, this app provides a way of collecting network information that will be exported to the edge cluster.
+this App Along with the Web UI Interface, provides a good view of the network topology and the devices that are connected to the devices network.
```

Comment:
Do you think it makes sense to add links to repos?

Reply:
I already added it at the end of the article, Do you think adding it inside the blogpost is more suitable?